Mathlib/MeasureTheory/Function/AEEqFun.lean

```diff
@@ -206,6 +206,69 @@ theorem induction_on₃ {α' β' : Type _} [MeasurableSpace α'] [TopologicalSpa
   induction_on f fun f hf => induction_on₂ f' f'' <| H f hf
 #align measure_theory.ae_eq_fun.induction_on₃ MeasureTheory.AEEqFun.induction_on₃
 
+/-!
+### Composition of an a.e. equal function with a (quasi) measure preserving function
+-/
+
+section compQuasiMeasurePreserving
+
+variable [MeasurableSpace β] {ν : MeasureTheory.Measure β} {f : α → β}
+
+open MeasureTheory.Measure (QuasiMeasurePreserving)
+
+/-- Composition of an almost everywhere equal function and a quasi measure preserving function.
+
+Many lemmas about this definition assume that `f` is a measure preserving map. -/
```

Comment:
This is not really true, as you use another definition for composition with measure preserving maps. Maybe add a pointer here to this other definition?

Reply:
Done.